README.md

```diff
@@ -3,10 +3,20 @@
 This is a high level API for using machine learning models in OpenMM simulations.  With just a few lines of code, you
 can set up a simulation that uses a standard, pretrained model to represent some or all of the interactions in a system.
 
-The current version is an early release.  The only supported potential functions are ANI-1ccx and ANI-2x.  They are
+In the current release, the only supported potential functions are ANI-1ccx and ANI-2x.  They are
```

Comment:
Add hyperlinks to these models:
```
* [ANI-1ccx](https://www.nature.com/articles/s41467-019-10827-4)
* [ANI-2x](https://pubs.acs.org/doi/full/10.1021/acs.jctc.0c00121)
```
I'd also link TorchANI, e.g.,
```
using optimized versions of the [pytorch](https://pytorch.org/) models available from [TorchANI](https://github.com/aiqm/torchani)
```
It wouldn't hurt to add an acknowledgment thanking the ANI developers for their help as well.

Reply:
Done.